### PR TITLE
Re-adds appendicitis

### PR DESCRIPTION
**Pull request by Alberyk**
This pr re-adds appendicitis and its random event, but this time it is all based on the organ, instead of a virus.
Also, makes the organ modification system in the character setup more robust. People can now start without an appendix.

### CODE REVIEW

**Comment by furrycactus** [comment]
Will we have the option to select 
Appendix: Existn't
in character setup, so that people can either opt out of it, or keep it canon if they get their appendix taken out in a canon round so it doesn't mysteriously strike again?

File: code/modules/events/spontaneous_appendicitis.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/datum/event/spontaneous_appendicitis
+	no_fake = 1
+
+/datum/event/spontaneous_appendicitis/start()
+	for(var/mob/living/carbon/human/H in shuffle(living_mob_list)) if(H.client && H.stat != DEAD)
```
Comment (Arrow768) [inline code/modules/events/spontaneous_appendicitis.dm:5]: Might be a good idea to check for antag status here.
Could be difficult for a antag to get that resolved if it happens mid round

Comment (Alberyk) [reply code/modules/events/spontaneous_appendicitis.dm:5]: Fixed.

File: code/modules/client/preference_setup/general/03_body.dm [hunk 2]
```diff
@@ -628,32 +637,31 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
...
+				pref.organ_data[organ_name] = null
 			if("Assisted")
-				pref.organ_data[organ] = "assisted"
+				pref.organ_data[organ_name] = "assisted"
 			if("Mechanical")
-				pref.organ_data[organ] = "mechanical"
+				pref.organ_data[organ_name] = "mechanical"
+			if("Removed")
+				pref.organ_data[organ_name] = "removed"
+		qdel(altered_organ)
```
Comment (Arrow768) [inline code/modules/client/preference_setup/general/03_body.dm:664]: Indentation issue

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — CHANGES_REQUESTED** [review]
The excel file for the event probabilities needs to be updated

**Comment by Alberyk** [comment]
The excel file was not updated when the event was removed, so no need to update it since the appendicitis info is there already.

**Review by Arrow768 — APPROVED** [review]

File: code/modules/client/preference_setup/general/03_body.dm [hunk 3]
```diff
@@ -245,6 +245,10 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 				organ_name = "liver"
 			if(BP_KIDNEYS)
 				organ_name = "kidneys"
+			if(BP_STOMACH)
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/general/03_body.dm:248]: This is getting extremely bloaty. At this point this should just use the species internal organs and go from there.

File: code/modules/organs/internal/appendix.dm [hunk 4]
```diff
@@ -2,4 +2,44 @@
...
+		if(inflamed > 600)
+			if(prob(1))
+				if(owner.can_feel_pain())
+					owner.custom_pain("You feel a stinging pain in your abdomen!")
+					owner.Weaken(10)
+
+				var/obj/item/organ/external/E = owner.get_organ(parent_organ)
+				E.sever_artery()
+				E.germ_level = max(INFECTION_LEVEL_TWO, E.germ_level)
+				owner.adjustToxLoss(25)
```
Comment (NonQueueingMatt) [inline code/modules/organs/internal/appendix.dm:43]: I'm not so sure about this. This is going to give damage to the liver directly iirc - or maybe the kidneys. Is this intended?

File: code/modules/organs/internal/appendix.dm [hunk 5]
```diff
@@ -2,4 +2,44 @@
...
+					owner.visible_message("<B>\The [owner]</B> winces painfully.")
+				owner.adjustToxLoss(1)
+		if(inflamed > 400)
+			if(prob(1))
+				germ_level += rand(2,6)
+				owner.vomit()
+		if(inflamed > 600)
+			if(prob(1))
+				if(owner.can_feel_pain())
+					owner.custom_pain("You feel a stinging pain in your abdomen!")
```
Comment (NonQueueingMatt) [inline code/modules/organs/internal/appendix.dm:37]: add a pain value after this

File: code/modules/organs/internal/appendix.dm (hunk 4 above)
Comment (Alberyk) [reply code/modules/organs/internal/appendix.dm:43]: Yes.

**Review by Geevies — APPROVED** [review]

**Pull request merged**